### PR TITLE
improved mobile layout

### LINKED ISSUE #16: Improve mobile layout
- [x] Add hamburger menu functionality to display a navigation mobile menu. So, the user can navigate different pages on mobile.
- [x] Also consider moving the menu out of the search box.
image

- [x] Improve card responsiveness on mobile layout.
image

### PR DESCRIPTION
**Pull request by Tomas2207**
Fixed hamburguer menu layout and improved images in customer service section.

### CODE REVIEW

File: src/components/CustomerService.js [hunk 1]
```diff
@@ -101,7 +101,8 @@ const CustomerService = () => {
           What would you like help with today? You can quickly take care of most
           things here, or connect with us when needed.
         </Typography>
-        <Grid container rowSpacing={1} columnSpacing={{ xs: 1, sm: 2, md: 3 }}>
+
```
Comment (wandergithub) [inline src/components/CustomerService.js:104]: Nice work with the new grid ✔️

File: .github/workflows/linters.yml [hunk 2]
```diff
@@ -6,14 +6,14 @@ env:
   FORCE_COLOR: 1
 
 jobs:
-  eslint:
```
Comment (wandergithub) [inline .github/workflows/linters.yml]: I'll respectfully undo these changes; I know it's not a big deal but I think keeping the full name of the lint would keep things clear.

File: src/App.css [hunk 3]
```diff
@@ -5,7 +5,7 @@ body {
 .app-container {
   display: flex;
   flex-direction: column;
-  width: 100vw;
+  /* width: 100vw; */
```
Comment (wandergithub) [inline src/App.css]: Yeah, I kind of noticed that this `width` property can be removed.

File: src/components/NavBar.js [hunk 4]
```diff
@@ -8,7 +8,7 @@ import { Link } from 'react-router-dom';
 import { useMediaQuery } from '@mui/material';
 
 const NavBar = () => {
-  if (useMediaQuery('(min-width:450px)')) {
+  if (useMediaQuery('(min-width:650px)')) {
```
Comment (wandergithub) [inline src/components/NavBar.js:11]: Nice catch. I agree with changing the query threshold. 👍🏽

File: src/style/customerService.css [hunk 5]
```diff
@@ -1,8 +1,18 @@
 .flex-row {
   display: flex;
-  flex-direction: row;
+  flex-direction: column;
   align-items: center;
```
Comment (wandergithub) [inline src/style/customerService.css]: Here is a CSS property duplicate, I'll remove it.

**Review by wandergithub — APPROVED** [review]
# All changes looks great ✔️ 

The new grid is responsive ✔️ 
Better layout ✔️

Thanks for your contribution @Tomas2207

**Comment by Tomas2207** [comment]
## Glad I could contribute with the project!

Hopefully my changes helped a bit

**Pull request merged**